src/app/public/modules/phone-field/phone-field.component.ts

```diff
@@ -116,13 +116,34 @@ export class SkyPhoneFieldComponent implements OnDestroy, OnInit {
 
       this.defaultCountryData = this.countries
         .find(country => country.iso2 === this._defaultCountry);
+
+      if (this.supportedCountryISOs && this.supportedCountryISOs.indexOf(this.defaultCountry) < 0) {
+        throw 'The `supportedCountryISOs` do not include the given `defaultCountry`!';
+      }
     }
   }
 
   public get defaultCountry(): string {
     return this._defaultCountry;
   }
 
+  /**
+   * Specifies the [International Organization for Standardization Alpha 2](https://www.nationsonline.org/oneworld/country_code_list.htm)
+   * country codes for the countries that users can select. By default, all countries are available.
+   */
+  @Input()
+  public set supportedCountryISOs(value: string[]) {
+    this._supportedCountryISOs = value;
+
+    if (this.defaultCountry && value && value.indexOf(this.defaultCountry) < 0) {
+      throw 'The `supportedCountryISOs` do not include the given `defaultCountry`!';
```

Comment:
Instead of throwing an error, what if you add the default country automatically if it doesn't exist in the `value` provided?

Reply:
(Same with the other error, above.)

Reply:
I've modified this to save the user supplied countries but also have a public version which include the default country in order to include that but also be able to react when the user changes the default country and didn't include it in the supported countries.